libs/dal/src/repositories/notification-template/notification-template.repository.ts

```diff
@@ -187,6 +187,68 @@ export class NotificationTemplateRepository extends BaseRepository<
 
     return { totalCount: totalItemsCount, data: this.mapEntities(items) };
   }
+  /**
+   * @deprecated This method is deprecated and will be removed in future versions.
+   * Use `getList` instead to retrieve notification templates without excluding new ones.
+   *
+   * @param {string} organizationId - The ID of the organization.
+   * @param {string} environmentId - The ID of the environment.
+   * @param {number} [skip=0] - The number of items to skip.
+   * @param {number} [limit=10] - The maximum number of items to return.
+   * @param {string} [query] - Optional query string to filter results.
+   * @returns {Promise<{ totalCount: number; data: NotificationTemplate[] }>} - A promise
+   * that resolves to an object containing the total count of items and the list of notification templates.
+   */
+  async getListExcludeNewDashboardWorkflows(
```

Comment:
Can we reuse `getList` with one more option for this to avoid code duplication?

Reply:
The code will stay around for a while so let's reuse to avoid changing one method but not the other.